tests/test_engineering_contract_docs.py

```
@@ -0,0 +1,55 @@
+    "docs/engineering/2026-05-07-framework-optimization-spec.md",
+    "docs/engineering/2026-05-07-skill-runner-contract.md",
+    "docs/engineering/2026-05-07-output-ownership-contract.md",
+    "docs/engineering/2026-05-07-alias-ownership-contract.md",
+    "docs/engineering/2026-05-07-bot-runner-contract.md",
+    "docs/engineering/2026-05-07-skill-help-contract.md",
+    "docs/engineering/domain-input-contracts.md",
+]
+
+
+def _is_tracked(relative_path: str) -> bool:
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)